### ISSUE TITLE
Z Button becomes stuck on the Nunchuck

### ISSUE THREAD

**Issue by GoogleCodeExporter** [javilechero/android-bluez-ime#95]
```
What steps will reproduce the problem?
1. Press the z button on the nunchuck
2.
3.

What is the expected output? What do you see instead?
Expected Z button operation. Instead the z-button becomes always pressed down. 
Holding the Z Button down causes the z button to act like it isn't pressed. 
Letting go once again the z button becomes stuck


What version of the product are you using? On what operating system?
Im using the current version that can be checked out from source. Im using an 
Asus Transformer on Honeycomb 3.2.1


Please provide any additional information below.
Im using my controller with N64oid


```

Original issue reported on code.google.com by `djLarri...@gmail.com` on 6 Oct 2011 at 12:18

**Comment by GoogleCodeExporter** [comment]
```
Please try the attached version, it should fix the issues with a stuck C and Z 
button as well as the inverse up/down for the nunchuck
```

Original comment by `kenneth@hexad.dk` on 9 Oct 2011 at 7:51
- Changed state: **Accepted**

Attachments:
- [Bluez-IME-v1.10-wiimote.apk](https://storage.googleapis.com/google-code-attachments/android-bluez-ime/issue-95/comment-1/Bluez-IME-v1.10-wiimote.apk)

**Comment by GoogleCodeExporter** [comment]
```
No feedback, I assume this is fixed in v1.10, let me know if that is not the 
issue.
```

Original comment by `kenneth@hexad.dk` on 12 Oct 2011 at 8:54

**Comment by GoogleCodeExporter** [comment]
Original comment by `kenneth@hexad.dk` on 12 Oct 2011 at 8:54
- Changed state: **Fixed**
